src/scenes/Organizations/Detail/OrganizationDetail.tsx

```diff
@@ -0,0 +1,70 @@
+import { Breadcrumbs, makeStyles, Typography } from '@material-ui/core';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Breadcrumb } from '../../../components/Breadcrumb';
+import { useDialog } from '../../../components/Dialog';
+import { PencilCircledIcon } from '../../../components/Icons';
+import { EditOrganizationForm } from './EditOrganization';
+import { useOrganizationQuery } from './OrganizationDetail.generated';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  root: {
+    flex: 1,
+    overflowY: 'scroll',
+    padding: spacing(4),
+    '& > *': {
+      marginBottom: spacing(3),
+    },
+  },
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  iconSpacing: {
+    marginRight: spacing(1),
+    cursor: 'pointer',
+  },
+}));
+
+export const OrganizationDetail = () => {
+  const classes = useStyles();
+  const { orgId } = useParams();
+
+  const { data, error } = useOrganizationQuery({
+    variables: {
+      input: orgId,
+    },
+  });
+
+  const [editOrgState, editOrg] = useDialog();
+
+  return (
+    <div className={classes.root}>
+      {error ? (
+        <Typography variant="h4">Error fetching Project</Typography>
+      ) : (
+        <>
+          <Breadcrumbs>
+            <Breadcrumb to="/organizations">Organizations</Breadcrumb>
+          </Breadcrumbs>
+          <Typography variant="h2">{data?.organization.name.value}</Typography>
+          <Typography
+            variant="body2"
+            color="primary"
+            className={classes.container}
+          >
+            <PencilCircledIcon
+              className={classes.iconSpacing}
+              onClick={editOrg}
+            />
+            Edit Organization
+          </Typography>
```

Comment:
Let's make this a button. Maybe it can go next to the org name? Also "Edit" or "Edit Partner"

Reply:
Done, I removed the button padding because it didn't look good but still more semantic.